### ISSUE TITLE
No id in ExchangeCreateContract

### ISSUE THREAD

**Issue by AngelQuirogaM** [tronprotocol/protocol#87]
Whenever the contract is created these are the parameters it returns:
{
	"ownerAddress": "TGAGGMBKK1fq2Ym3ZQDzTWa55R68UB5xWU",
	"firstTokenId": "SEED",
	"firstTokenBalance": 1500,
	"secondTokenId": "_",
	"secondTokenBalance": 7500000000,
	"contractType": 41,
	"hash": "54d41e4408e74de3ec9d8cf25362676f41e37a72d8dcb05b09ce10db603c5a6f",
	"time": 1541743302279
}

...but it lacks the most important one which is the 'exchangeId'. with it we can get track the secondary token which is not in the ExchangeTransactionContract.

**Comment by Kondax** [comment]
I agree, this should definitely be a parameter of the contract. The three major issues:
1. IDs need to be unique
2. IDs need to be "cheap" as each exchange call will depend on the ID
3. They should be able to be calculated offline

Each issue is easy to overcome independently, just not when you consider all three.

One solution I propose is using the `blockRef & index`. This means both issues 2 and 3 can be safely assumed to not be overcome, whereas issue 1 is minimal that it will not be harmful. The index doesn't have to be enforced to be incremental either - as long as it's suitable (current milliseconds into the current second would be suitable in my opinion).
